### PR TITLE
Added support for reverse ranges.

### PR DESCRIPTION
**Pull request by smith-sj**
Duplicated code in the ternary statement, could abstract out into a function but I think this reads okay..

### CODE REVIEW

**Review by Yearnsmith — CHANGES_REQUESTED** [review]
Looks like a good start. :)

>  could abstract out into a function

It's already kind of abstracted... so it'd be a matter of refactoring `mkr` to derive variables from the parameters, like `big`, `small`, and `direction`. That way we could minus `big` from `small`, and plus or minus based on `direction`. Though this is probably adding too much complexity? :shrug: 

Otherwise, I think I can approve this if you fix the following:
1. For consistency in style, it's probably best to use `_e` and `_s` in place of `end` and `start`
2. For readability, please format the ternary as:
    ```js
    query
      ? trueExpression
      : falseExpression
    ```

edit: formatting

**Review by Yearnsmith — APPROVED** [review]

**Comment by smith-sj** [comment]
> That way we could minus big from small, and plus or minus based on direction. Though this is probably adding too much complexity?

Yeah, good point. I think it might make it more complex than it needs to be. Your call though, I'd be happy to do it.

**Comment by Yearnsmith** [comment]
I might have to go back and revisit that function for the other features, anyway.

For now I'm happy it's doing the reverse thing, And I can reproduce your logic in any future revisions. :+1:

**Pull request merged**
